apps/intellij/src/main/kotlin/dev/nx/console/graph/NxGraphBrowserBase.kt

```diff
@@ -321,8 +325,9 @@ abstract class NxGraphBrowserBase(protected val project: Project) : Disposable {
               }
             </style>
 
-            <p>Unable to load the project graph. The following error occurred:</p>
-      <pre>${error}</pre>
+            <p>Unable to load the project graph. The following error${if(errors.isNotEmpty()) "s" else ""} occurred:</p>
+      ${errors.map { "<pre>${it.message ?: ""} \n ${it.stack ?: ""}</pre>" }.joinToString("\n")
+        }
       If you are unable to resolve this issue, click here to <a href="#" onclick="window.reset()">reset</a> the graph.
```

Comment:
Should we also say to run `nx reset` in the terminal as an alternative? In the case of the `CancellationException` it is the only way to resolve.